src/app/routes/onDemandAudio/getInitialData/index.ts

```diff
@@ -117,6 +119,14 @@ export default async ({
       ? await getPodcastExternalLinks(service, brandId, variant, versionId)
       : [];
 
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const mediaBlocks = get(['content', 'blocks']).map((block: any) => {
+      return {
+        type: 'onDemandAudio',
+        model: { ...block },
+      };
+    });
```

Comment:
**Note to reviewers:** Adding media blocks to the pageData response, in preparation for replacing the legacy media player with the media loader.

Note that this will increase the page weight slightly, as we are duplicating the data. However this will be temporary until we add the media loader & then delete the data which is no longer required